release/src/main/python-release/python_release_automation_utils.sh

```diff
@@ -218,9 +218,10 @@ function create_pubsub() {
 #   None
 #######################################
 function cleanup_pubsub() {
-  gcloud pubsub topics delete --project=$PROJECT_ID $PUBSUB_TOPIC1
-  gcloud pubsub topics delete --project=$PROJECT_ID $PUBSUB_TOPIC2
-  gcloud pubsub subscriptions delete --project=$PROJECT_ID $PUBSUB_SUBSCRIPTION
+  # Suppress error since topic/subscription may not exist
+  gcloud pubsub topics delete --project=$PROJECT_ID $PUBSUB_TOPIC1 2> /dev/null
```

Comment:
Also suppress standard output?

Reply:
There is no message in stdout by calling `gcloud pubsub topics create/delete` (whether succeeds or not).

Reply:
If succeed, it will show messages like 
**"Created topic [projects/apache-beam-testing/topics/wordstream-python-topic-1]."
"Deleted topic [projects/apache-beam-testing/topics/wordstream-python-topic-1]."**
If fail, it will raise something like **"ERROR: The topic XXXX does not exist".**

Reply:
right, those messages go to standard error.

Reply:
Also the main purpose to suppress stderr here is not to break the script. Messages in stdout can help people know the status of current step.